### PR TITLE
PLG-692 : Update email reset password

### PR DESCRIPTION
**Pull request by antoinetrouve**
**Description (for Contributor and Core Developer)**

See Ticket : https://akeneo.atlassian.net/jira/software/projects/PLG/boards/252?selectedIssue=PLG-692

Waiting [PR 16077](https://github.com/akeneo/pim-community-dev/pull/16077) be merged.

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: src/Akeneo/Platform/Bundle/NotificationBundle/Resources/views/Email/notification.html.twig [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            <table style="border: none; border-spacing: 0; border-collapse: collapse; vertical-align: top;">
+                <tbody style="border: none; border-spacing: 0; border-collapse: collapse; vertical-align: top;">
+
+                    <!-- header -->
+                    <tr style="border: none;border-spacing: 0;border-collapse: collapse;vertical-align: top;">
+                        <td style="border: none;border-spacing: 0;border-collapse: collapse;vertical-align: top;padding-bottom: 20px;">
+                            {% block emailHeader %}
+                                <img style="width: 100%; height: auto; max-width: 242px; max-height: 59px;"
+                                     alt="Akeneo logo"
+                                     src="{{ akeneo_pim_url }}{{ asset('/bundles/pimnotification/images/email/logo.png') }}">
```
Comment (BryanS29-akeneo) [inline src/Akeneo/Platform/Bundle/NotificationBundle/Resources/views/Email/notification.html.twig:25]: We can concatanate like `{{ akeneo_pim_url ~ asset('bundles/pimnotification/images/email/logo.png') }}`

File: src/Akeneo/Platform/Bundle/NotificationBundle/Resources/views/Email/notification.html.twig [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                                <div style="font-size: 9px;">
+                                    <p>Copyright © {{ 'now' | date('Y') }} Akeneo, All rights reserved.</p>
+                                    <p>
+                                        You have been invited to use Akeneo Shared Catalogs.
+                                        As it is not direct marketing emails, you cannot unsubscribe to this email.
+                                    </p>
+                                </div>
+                                <img style="width: 100%; height: auto; max-height: 32px; max-width: 131px; margin-top: 15px;"
+                                     alt="Akeneo signature"
+                                     src="{{ akeneo_pim_url }}{{ asset('bundles/pimnotification/images/email/signature.png') }}">
```
Comment (BryanS29-akeneo) [inline src/Akeneo/Platform/Bundle/NotificationBundle/Resources/views/Email/notification.html.twig:64]: We can concatanate like `{{ akeneo_pim_url ~ asset('bundles/pimnotification/images/email/signature.png') }}`

File: src/Akeneo/Tool/Bundle/BatchBundle/Resources/views/Email/notification.html.twig [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+{% block emailUserAction %}
+    {% if akeneo_pim_url is not empty %}
+        <span>
+            <a target="_blank"
+               role="button"
+               style="-webkit-appearance: button; -moz-appearance: button; appearance: auto;
+               padding: 10px 20px; border-radius: 25px; background-color: #9452ba; color: white;
+               text-decoration: none; text-transform: uppercase; font-size: 15px; font-weight: 900;"
+               href="{{ akeneo_pim_url }}/#{{ path('akeneo_job_process_tracker_details', {'id': jobExecution.id}) }}">View summary</a>
```
Comment (BryanS29-akeneo) [inline src/Akeneo/Tool/Bundle/BatchBundle/Resources/views/Email/notification.html.twig:62]: We can concatanate like `{{ akeneo_pim_url ~ '/#' ~ path('akeneo_job_process_tracker_details', {'id': jobExecution.id}) }}`

File: src/Akeneo/Tool/Bundle/BatchBundle/Resources/views/Email/notification.txt.twig [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    Ended on {{ jobExecution.endTime|date("Y-m-d") }} at {{ jobExecution.endTime|date("H:i:s")  }}.
+
+    Results:
+    {% for stepExecution in jobExecution.stepExecutions %}
+      - {{ stepExecution.stepName|trans }} [{{ stepExecution.status }}]
+    {% endfor %}
+
+    {% if akeneo_pim_url is not empty %}
+        You can view full summary at :
+        {{ akeneo_pim_url }}/#{{ path('akeneo_job_process_tracker_details', {'id': jobExecution.id}) }}
```
Comment (BryanS29-akeneo) [inline src/Akeneo/Tool/Bundle/BatchBundle/Resources/views/Email/notification.txt.twig:15]: We can concatanate like `{{ akeneo_pim_url ~ '/#' ~ path('akeneo_job_process_tracker_details', {'id': jobExecution.id}) }}`

File: src/Akeneo/Platform/Bundle/NotificationBundle/Resources/views/Email/notification.html.twig (hunk 1 above)
Comment (antoinetrouve) [reply src/Akeneo/Platform/Bundle/NotificationBundle/Resources/views/Email/notification.html.twig:25]: Modification effectuée dans la [PR 16077](https://github.com/akeneo/pim-community-dev/pull/16077)

File: src/Akeneo/Platform/Bundle/NotificationBundle/Resources/views/Email/notification.html.twig (hunk 2 above)
Comment (antoinetrouve) [reply src/Akeneo/Platform/Bundle/NotificationBundle/Resources/views/Email/notification.html.twig:64]: Modification effectuée dans la [PR 16077](https://github.com/akeneo/pim-community-dev/pull/16077)

File: src/Akeneo/Tool/Bundle/BatchBundle/Resources/views/Email/notification.html.twig (hunk 3 above)
Comment (antoinetrouve) [reply src/Akeneo/Tool/Bundle/BatchBundle/Resources/views/Email/notification.html.twig:62]: Modification effectuée dans la [PR 16077](https://github.com/akeneo/pim-community-dev/pull/16077)

File: src/Akeneo/UserManagement/Bundle/Notification/MailResetNotifier.php [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Security\Core\User\UserInterface;
+use Throwable;
+use Twig\Environment;
+
+/**
+ * @copyright 2022 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class MailResetNotifier
```
Comment (LaurentPetard) [inline src/Akeneo/UserManagement/Bundle/Notification/MailResetNotifier.php:17]: Please add a unit test for this class (phpspec)

File: src/Akeneo/Tool/Bundle/BatchBundle/Resources/views/Email/notification.txt.twig (hunk 4 above)
Comment (antoinetrouve) [reply src/Akeneo/Tool/Bundle/BatchBundle/Resources/views/Email/notification.txt.twig]: Done .

File: src/Akeneo/UserManagement/Bundle/Notification/MailResetNotifier.php (hunk 5 above)
Comment (antoinetrouve) [reply src/Akeneo/UserManagement/Bundle/Notification/MailResetNotifier.php:17]: Done

**Review by LaurentPetard — APPROVED** [review]

**Review by BryanS29-akeneo — APPROVED** [review]

**Pull request merged**